openwisp_controller/config/static/config/js/widget.js

```diff
@@ -779,3 +754,29 @@ JSONEditor.defaults.themes.django = JSONEditor.AbstractTheme.extend({
         link.appendChild(image);
     }
 });
+
+// Override setValue method to allow using variables for fields with maxLength
+JSONEditor.defaults.editors.multiple.prototype.setValue = function (val, initial) {
+    // Determine type by getting the first one that validates
+    var self = this;
+    window._$each(this.validators, function (i, validator) {
```

Comment:
why are you using `window._$each` here and `forEach` below? Is it necessary or can we stick to one way of doing it?

Reply:
I have copied the code for helper function $each from the JSON Schema library:

https://github.com/openwisp/openwisp-controller/blob/c6a4c18488fe551ce536f568568793f1d5df569d/openwisp_controller/config/static/config/js/widget.js#L758-L778

`$each` do more than just iterating over all the values and I don't want to break the library because of our custom code.